source/backend/api/Helpers/Healthchecks/PimsLtsaHealthcheck.cs

```diff
@@ -0,0 +1,38 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Pims.Core.Exceptions;
+using Pims.Ltsa;
+
+namespace Pims.Api.Helpers.Healthchecks
+{
+    public class PimsLtsaHealthcheck : IHealthCheck
+    {
+        private readonly int _pid;
+        private readonly ILtsaService _ltsaService;
+
+        public PimsLtsaHealthcheck(IConfiguration configuration, ILtsaService ltsaService)
+        {
+            _pid = int.Parse(configuration.GetSection("HealthChecks:Ltsa:Pid").Value);
+            _ltsaService = ltsaService;
+        }
+
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
+        {
+            try
+            {
+                var titleSummary = await _ltsaService.GetTitleSummariesAsync(_pid);
```

Comment:
This is the lowest impact rest endpoint ltsa has - the only one that doesn't involve generating a document as a result of the request.